src/scripts/services/user-service.ts

```diff
@@ -196,6 +195,47 @@ export class OktaUserService {
           error
         )}].`
     );
+
+  readonly activateUser = (userId: string): TE.TaskEither<string, User> =>
+    TE.tryCatch(
+      () =>
+        // eslint-disable-next-line functional/no-this-expression, @typescript-eslint/prefer-readonly-parameter-types
+        this.client.getUser(userId).then((user) =>
+          user
+            .activate({
+              sendEmail: false,
```

Comment:
This is a good default, but it'd be nice if we could specify whether to send the activation email to the user. In many cases, we will want to.

Reply:
I reckon we do this as a new ticket. @vuthikxkol can you create a ticket for this and link it to @dbalmain so that he can resolve this commetn.

Reply:
@dspasojevic @dbalmain  I've created the ticket. Link: https://agiledigital.atlassian.net/browse/IE-339